test/acceptance/features/readyServiceBinding.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Ready Service Binding Should Be Read-Only
```

Comment:
Merge feature files in one, and rename feature to "Successful Service Binding Resources are Immutable".